### ISSUE TITLE
[bug] hash mismatch of downloaded file on v0.3.0/nsis_semvercompare.dll

### ISSUE THREAD

**Issue by kornelski** [tauri-apps/tauri#6128]
### Describe the bug

I've installed tauri-cli from the latest git `cargo install tauri-cli --git https://github.com/tauri-apps/tauri` and it fails to validate a file it downloads during the build:

```text
…
 Downloading https://github.com/tauri-apps/nsis-semvercompare/releases/download/v0.3.0/nsis_semvercompare.dll
        Info validating hash
       Error failed to bundle project: hash mismatch of downloaded file
```

### Reproduction

`cargo tauri build`
…
```text
        Warn NSIS directory is missing some files. Recreating it.
        Info Verifying NSIS package
 Downloading https://sourceforge.net/projects/nsis/files/NSIS%203/3.08/nsis-3.08.zip/download
        Info validating hash
        Info extracting NSIS
 Downloading https://github.com/tauri-apps/binary-releases/releases/download/nsis-plugins-v0/NScurl-1.2022.6.7.zip
        Info extracting NSIS NScurl plugin
 Downloading https://github.com/tauri-apps/binary-releases/releases/download/nsis-plugins-v0/NSIS-ApplicationID.zip
        Info extracting NSIS ApplicationID plugin
 Downloading https://github.com/tauri-apps/binary-releases/releases/download/nsis-plugins-v0/NsProcess.zip
        Info extracting NSIS NsProcess plugin
 Downloading https://github.com/tauri-apps/nsis-semvercompare/releases/download/v0.3.0/nsis_semvercompare.dll
        Info validating hash
       Error failed to bundle project: hash mismatch of downloaded file
```

### Expected behavior

The same command works with 2.0.0-alpha.1

### Platform and versions

Environment
  › OS: Windows 10.0.19044 X64
  › Webview2: 109.0.1518.61
  › MSVC:
      - Visual Studio Community 2019
  › Node.js: Not installed!
  › npm: Not installed!
  › pnpm: Not installed!
  › yarn: Not installed!
  › rustup: 1.25.1
  › rustc: 1.66.1
  › cargo: 1.66.1
  › Rust toolchain: stable-x86_64-pc-windows-msvc

Packages
WARNING: no lock files found, defaulting to npm
  › @tauri-apps/cli [NPM]: 1.2.3
  › @tauri-apps/api [NPM]: Not installed!
  › tauri [RUST]: 1.2.3,
  › tauri-build [RUST]: 1.2.1,
  › tao [RUST]: 0.15.6,
  › wry [RUST]: 0.23.4,

**Comment by amrbashir** [comment]
this will be fixed by #6120

**Comment by amrbashir** [comment]
for now you can install the cli from this commit 4892637f83b0f64822a5d304e8786a4c92a8957d
